src/index.html

```diff
@@ -12,7 +15,55 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
+    <div class="stars-page">
+      <div class="stars page__item stars--0">
```

Comment:
Remove this class everywhere 
```suggestion
      <div class="stars stars--0">
```